minetest/minetest
# Fix crash if calling minetest.remove_player on startup time

This PR fixes a crash if you called `minetest.remove_player` too early (before server environment is ready).

If the function is called too early, Minetest will no longer crash but will return the number 3, which stands for failure because the server was not initialized yet.

## To do

This PR is ready for Review.

## How to test

Start any game with this minimal test mod:
```
minetest.log("none", "RETURN="..minetest.remove_player("foobar"))
```

In console, you should see "RETURN=3".
